### DISCUSSION TITLE
是否可支援 Node.JS ?

### DISCUSSION THREAD

**Discussion by JeffKko** [sml2h3/ddddocr#116] [General]
首先 謝謝 `sml2h3` 大佬寫了這麼牛B的 library

小弟對 OCR, Deep learning 不是很了解

但有看到 Node.js 也有個 [onnxruntime](https://www.npmjs.com/package/onnxruntime-node) 套件

是不是代表在 Node.js 上也可以使用 onnx 模型, 進而做到和 ddddocr 一樣的效果?

感謝 🙏

**Comment by renhaoyeh** [comment]
我有用 TypeScript 實現了
以下為相關連結

[Github](https://github.com/rhy3h/ddddocr-node)
[NPM](https://www.npmjs.com/package/ddddocr-node)
